DynamicRouter.php

```diff
@@ -166,17 +167,17 @@ public function getGenerator()
      *
      * @api
      */
-    public function generate($name, $parameters = array(), $referenceType = false)
+    public function generate($route, $parameters = array(), $referenceType = false)
```

Comment:
i don't want to rename this here, as its implementing the RouterInterface which calls the thing `$name`